compiler/ccgexprs.nim

```diff
@@ -413,6 +413,8 @@ proc genAssignment(p: BProc, dest, src: TLoc, flags: TAssignmentFlags) =
   of tyPtr, tyPointer, tyChar, tyBool, tyEnum, tyCstring,
      tyInt..tyUInt64, tyRange, tyVar, tyLent, tyNil:
     linefmt(p, cpsStmts, "$1 = $2;$n", [rdLoc(dest), rdLoc(src)])
+  of tyUncheckedArray:
+    localError(p.config, dest.lode.info, "assignment of UncheckedArray is not supported with ARC/ORC")
```

Comment:
It never was supported, this is not ARC/ORC specific!

Reply:
it compile and runs on `refc`, and `refc` doesn't goes in here.

Reply:
it only happened to work with the other GCs. It was never supported as it never made sense.

Reply:
so what's the error message would like